aten/src/ATen/functorch/DynamicLayer.h

```diff
@@ -15,6 +15,8 @@
 #include <ATen/functorch/ADInterpreters.h>
 #include <ATen/functorch/FunctionalizeInterpreter.h>
 
+#include <nlohmann/json.hpp>
```

Comment:
Do you need to serialize this in C++?

Reply:
Yeah I don't need this header in this file anymore (leftover from the previous version).